packages/cli-kit/src/log.ts

```diff
@@ -120,11 +138,12 @@ function getLogFilePath(options: {logDir?: string; override?: boolean} = {}) {
 
 // Shaves off older log lines if logs are over maxLogFileSize long.
 async function truncateLogs(logFile: string) {
-  if (fileSizeSync(logFile) < maxLogFileSize) {
+  const fileSize = fileSizeSync(logFile)
+  if (fileSize < maxLogFileSize) {
     return
   }
   const tmpLogFile = logFile.concat('.tmp')
-  const truncateLines = new LinesTruncatorTransformer(maxLogFileSize)
+  const truncateLines = new LinesTruncatorTransformer(fileSize)
   const pipeline = promisify(Stream.pipeline)
   await pipeline(createReadStream(logFile), truncateLines, createWriteStream(tmpLogFile))
   await pipeline(createReadStream(tmpLogFile), createWriteStream(logFile))
```

Comment:
In the interest of further performance improvement, it feels like this should be a copy/move instead of reading a file just to write it elsewhere.

If this was `await moveFile(tmpLogFile, logFile)` we could also skip the `unlinkSync` below.

Reply:
The `tmpLogFile` is the result of the truncation process so its size should be 5MB at maximum. With that size I don't think that we have a better performance switching from this `pipeline` to a `move` operation, what do you think?